src/modules/configurator/components/const/months.ts

```diff
@@ -0,0 +1 @@
+export const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
```

Comment:
Constants should have a type, when you have a predefined list you can just put `as const` at the end, and it will behave like union type

Reply:
Removed months.ts file completely and implemented date-fns for formatting.